cfgov/unprocessed/css/organisms/secondary-navigation.less

```diff
@@ -0,0 +1,161 @@
+                <ul class="o-secondary-navigation_list
+                           o-secondary-navigation_list__parents">
+                    <li class="o-secondary-navigation_item
+                               o-secondary-navigation_item__parent">
+                        nav-link
+                    </li>
+                    <li class="o-secondary-navigation_item
+                               o-secondary-navigation_item__parent">
+                        nav-link
+                    </li>
+                    <li class="o-secondary-navigation_item
+                               o-secondary-navigation_item__parent">
+                        nav-link
+                    </li>
+                </ul>
+            </div>
+        </nav>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .expandable.o-secondary-navigation
+            .expandable_header
+              .expandable_target.o-secondary-navigation_button
+                .expandable_header-left
+                .expandable_header-right
+                  .expandable_cue-open
+                    .cf-icon.cf-icon-down
+                  .expandable_cue-close
+                    .cf-icon.cf-icon-up
+            .expandable_content
+              .o-secondary-navigation_list.o-secondary-navigation_list__parents
+                .o-secondary-navigation_item.o-secondary-navigation_item__parent
+  tags:
+    - cfgov-organisms
+*/
+
+.o-secondary-navigation {
+    .webfont-regular();
+
+    .respond-to-max(@bp-sm-max, {
+        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
+
+        background-color: @gray-10;
+        border-top: 1px solid @gray-20;
+        border-bottom: 1px solid @gray-20;
+    });
+
+    &_button {
+        width: 100%;
+        padding: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+
+        .webfont-demi();
+
+        text-align: left;
+        text-transform: uppercase;
+
+        &:focus {
+            outline: 1px dotted;
+        }
+
+        .cf-icon {
+            color: @green;
+        }
+
+        .respond-to-min(@bp-med-min, {
+            display: none;
+        });
+    }
+
+    &_list {
+        margin: 0;
+        padding: 0;
+
+        &__parents {
+            .respond-to-max(@bp-sm-max, {
+                padding-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+                margin-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+                margin-right: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+
+                border-top: 1px solid @gray-50;
+            });
+        }
+
+        &__children {
+            margin-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+
+            .respond-to-min(@bp-med-min, {
+                // Add 5px for the border to half the gutter
+                margin-left: unit( (@grid_gutter-width / 2 + 5px ) / @base-font-size-px, em) ;
```

Comment:
I think the order of operations is okay here but just flagging to ensure this isn't ending up `(@grid_gutter-width / 7px )`